### DISCUSSION TITLE
Wyze Cam v3 [WZ_Mini Hacks] Disabling OSD (date & time)

### DISCUSSION THREAD

**Discussion by LyrixCaz** [gtxaspec/wz_mini_hacks#724] [Q&A] [answered]
How can I disable the date/time on screen display? I have frigate and it sometimes mistakens the seconds changing as motion, How can I disable it?

**Comment by amandamana** [comment, marked as answer]
If your cam is online, then turn off "Show Timestamp" under advanced settings in the app. If it's not connected to the cloud, then you need add the line `osdSwitch=2`  in the `[SETTING]` section of the `/configs/.user_configs` file.

**Comment by jmbulan** [comment]
I choose to leave mine on and just apply a motion filter in frigate. If your on . 13  it got even simpler. Just head over to cameras then debug and you can add zones and mask and save them with out having to do the copy paste thing before. I chose to leave it on so I know the fees hasn't gotten stuck or hung up. If anyone has a better way of verifying the fees is fact live, I'm open to ideas.

**Reply by LyrixCaz** [reply]
I ended up doing the same, as adding a timestamp via frigate would use some resources.
